rocketpy/plots/rocket_plots.py

```diff
@@ -1,8 +1,63 @@
-__author__ = " "
+__author__ = "Oscar Mauricio Prada Ramirez"
 __copyright__ = "Copyright 20XX, RocketPy Team"
 __license__ = "MIT"
 
 
+import matplotlib.pyplot as plt
+import numpy as np
+
+
 class _RocketPlots:
-    def __init__(self) -> None:
-        pass
+    """Class that holds plot methods for Environment class.
+
+    Attributes
+    ----------
+    _RocketPlots.rocket : Rocket
+        Rocket object that will be used for the plots.
+
+    """
+
+    def __init__(self, rocket) -> None:
+        """Initializes _EnvironmentPlots class.
+
+        Parameters
+        ----------
+        rocket : Rocket
+            Instance of the Rocket class
+
+        Returns
+        -------
+        None
+        """
+
+        self.rocket = rocket
+
+        return None
+
+    def powerOnDrag(self):
+
+        self.rocket.powerOnDrag()
+
+        return None
```

Comment:
I think we need a method like this for every other plot that is in `def all(self)`. It makes it more organized and makes things easier if we ever want to further customize plots in the future.

Also, simple documentation on these new methods is necessary